eva_submission/eload_validation.py

```diff
@@ -353,6 +358,93 @@ def _collect_validation_workflow_results(self, output_dir):
             })
         self.eload_cfg.set('validation', 'assembly_check', 'pass', value=total_error == 0)
 
+    def _detect_structural_variant(self):
```

Comment:
Suggest extracting this to a separate file for better readability...

Reply:
This will likely be done in a separate ticket